packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/constants.js.hbs

```diff
@@ -0,0 +1,18 @@
+// it's necessary that the interface remain consistent, as other
+// files in the base template rely on constants.js, thus we
+// import the underlying constants.js and re-export it
+
+import {
+    DEFAULT_LIMIT_VALUES,
+    DEFAULT_SEARCH_PARAMS
+} from '{{preset.templateSource.id}}/app/constants'
+
+// original value is 25
+DEFAULT_LIMIT_VALUES[0] = 3
+DEFAULT_SEARCH_PARAMS.limit = 3
```

Comment:
I'm not sure if these are good example to have in the generated project. I'm worried that it won't be very obvious to developers that these are the settings control the number of items on PLP. We might get an influx of questions "why is my generated project PLP only shows 3 items and how do i change it?" and we will keep answering "update your constant.js".

How do we keep a sane default and at the same time demonstrate how to extend files? 🤔

Reply:
Interesting point. I'm not sure, though I suppose one might say that we **want** to encourage these kinds of questions, they accelerate the learning curve. The "degree of ownership" to build a project successfully requires that they master these concepts. Putting this as a "front loaded" task that forces them to learn in the beginning I think will probably prevent lots of other mistakes later on (by copy / pasting the file and then breaking the API contract, at a minimum)